<launch>
  <node name="kinematic_services" type="kinematic_services" pkg="robot_kinematic_services" output="screen">
    <rosparam file="$(find robot_kinematic_services)/config/solver.yaml" command="load"/>
  </node>
</launch>
